tokens/eth/0x6BFf2fE249601ed0Db3a87424a2E923118BB0312.json

```diff
@@ -0,0 +1,15 @@
+{
+    "symbol": "FYZ",
+    "name": "Fyooz",
+    "address": "0x6BFf2fE249601ed0Db3a87424a2E923118BB0312",
+    "decimals": 18,
+    "website": "https://www.fyooz.io/",
+    "logo": "https://fyooz-miscellaneous.s3.eu-central-1.amazonaws.com/logos/FYOOZ_Coin-Icon_White-on-Black_128x128.png",

```

Comment:
sorry - did not see this when looking over it - but the CI found it - logo must be an object - see the other tokens for a reference